doc/api-docs/conf.py

```diff
@@ -235,45 +231,33 @@ class MockQGroupBox:
 EXCLUDED_MODULES = [
     '_core',  # anything ending in '_core'
     'recmodel_server',
-    'recmodel_workspace_fetcher'
+    'recmodel_workspace_fetcher',
+    'natcap.invest.ui'
```

Comment:
Based on the RTD-rendered docs (https://invest--990.org.readthedocs.build/en/990/api.html for this PR) it looks like maybe this exclusion isn't working.  Maybe it never worked?  In any case, it would probably be nice to exclude packages, or else just have everything in there since it _is_ in the source code.

Reply:
I think this is working as expected - it's excluding those from the model entrypoints file `models.rst`. But they're still in the API docs.
The list of excluded modules was unnecessary, though. I refactored so that any module with an `ARGS_SPEC` is included in `models.rst`.